device-types/Fortinet/FG-60D.yml

```diff
@@ -0,0 +1,36 @@
+---
+manufacturer: Fortinet
+model: FortiGate 60D
+slug: fg-60d
+part_number: FG-60D
+u_height: 1
+is_full_depth: false
+console-ports:
+  - name: Console
+    type: rj-45
+power-ports:
+  - name: PS1
+    type: nema-5-15p
```

Comment:
It looks like the `60D` is a `dc-terminal` not a `nema-5-15p`. Please adjust this accordingly

Reply:
you're right, I'll fix it